### PR TITLE
feat: additional service method options (CMC-790)

### PR DESCRIPTION
**Pull request by swalker125**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CMC-790

### Change description ###

New method of service options, left at permitted address and personal service
New field to get who the documents were served to

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by swalker125** [comment]
Moved to draft as unsure if certificate of service also requires changes due to this PR

File: ccd-definition/CaseField.json [hunk 1]
```diff
@@ -181,6 +181,16 @@
     "_Definition":  "The delivery method type of service from the applicant solicitor to the first respondent solicitor",
     "_Category":  "CIVIL"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "UNSPECIFIED_CLAIMS",
+    "ID": "serviceNamedPersonToRespondentSolicitor1",
+    "Label": "On whom did you serve?",
```
Comment (dharmendrak) [inline ccd-definition/CaseField.json:188]: is it On or To?

**Review by dharmendrak — DISMISSED** [review]

File: ccd-definition/CaseField.json (hunk 1 above)
Comment (swalker125) [reply ccd-definition/CaseField.json:188]: On apparently! Seems odd to me too 🤷 

"Who did you serve to?" sounds more correct imo

Comment (swalker125) [reply ccd-definition/CaseField.json:188]: maybe even name of person served to

**Review by pd-dudkiewicz — DISMISSED** [review]

File: ccd-definition/CaseField.json (hunk 1 above)
Comment (ckalista) [reply ccd-definition/CaseField.json:188]: I though it was only me because not native english speaker...

**Review by ckalista — DISMISSED** [review]

**Pull request merged**